sdk/cognitiveservices/azure-cognitiveservices-language-textanalytics/azure/cognitiveservices/language/textanalytics/__init__.py

```diff
@@ -1,18 +1,199 @@
 # coding=utf-8
-# --------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-#
-# Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is
-# regenerated.
-# --------------------------------------------------------------------------
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
 
-from .text_analytics_client import TextAnalyticsClient
-from .version import VERSION
+from typing import Any, Optional, List  # pylint: disable=unused-import
+from ._text_analytics_client import TextAnalyticsClient
+from ._response_handlers import _validate_single_input, process_single_error, process_entities_error
+from ._version import VERSION
+from ._models import (
+    LanguageInput,
+    MultiLanguageInput,
```

Comment:
In the .NET API, we're renaming some of these types.  It would be good to consider aligning these before a first preview.